<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_box_login">
     <property name="title">
      <string>Введите свой логин</string>
     </property>
     <layout class="QHBoxLayout" name="horizontal_layout">
      <item>
       <widget class="QLabel" name="label_login">
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_edit_login">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_set_login">
        <property name="text">
         <string>Задать логин</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontal_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_box_players">
     <property name="title">
      <string>Выберите противника</string>
     </property>
     <layout class="QVBoxLayout" name="vertical_layout">
      <item>
       <widget class="QTableWidget" name="table_widget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontal_layout_2">
        <item>
         <spacer name="horizontal_spacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="button_select_player">
          <property name="text">
           <string>Выбрать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_cancel">
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
